api/integrations/lead_tracking/hubspot/lead_tracker.py

```diff
@@ -68,7 +68,10 @@ def update_company_active_subscription(
     def create_user_hubspot_contact(self, user: FFAdminUser) -> str | None:
         tracker = HubspotTracker.objects.filter(user=user).first()
         tracker_cookie = tracker.hubspot_cookie if tracker else None
-        self.client.create_lead_form(user=user, hubspot_cookie=tracker_cookie)
+        utm_data = tracker.utm_data if tracker else None
+        self.client.create_lead_form(
+            user=user, hubspot_cookie=tracker_cookie, utm_data=utm_data
+        )
```

Comment:
Again, this is quite explicit and works just fine, so I'm not mandating that we change anything here. If we want to though, we can remove one of the independent null checks on the `tracker` by 'simplifying' it to something like: 

```python
        create_lead_form_kwargs = {"user": user}
        if tracker := HubspotTracker.objects.filter(user=user).first():
            create_lead_form_kwargs.update(
                {"utm_data": tracker.utm_data, "hubspot_cookie": tracker.hubspot_cookie}
            )
        self.client.create_lead_form(**create_lead_form_kwargs)
```